staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4453,7 +4455,11 @@ type ServiceAccount struct {
 	// can be mounted in the pod, but ImagePullSecrets are only accessed by the kubelet.
 	// More info: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
 	// +optional
-	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" protobuf:"bytes,3,rep,name=imagePullSecrets"`
+	// +patchMergeKey=name
+	// +patchStrategy=merge
```

Comment:
This addition is the bit that is incompatible with GA API functionality and needs more consideration.